.alexrc

```diff
@@ -0,0 +1,3 @@
+{
+  "allow": ["execute", "host-hostess", "remains", "reject", "traps", "invalid", "period", "invalid", "execution", "simple", "failure"]
```

Comment:
"invalid" is in there twice. Can you remove the duplicate entry, as well as rename the branch source branch to begin with the Jira ticket (CORTX-28870) for a cleaner collaboration history?

Specifics on how we want to foster contribution to the repo can be found in the https://github.com/Seagate/cortx-k8s/blob/integration/CONTRIBUTING.md document

Reply:
@osowski I have addressed your comments. However, I'm not able to rename the branch name in the same PR. So, I have created new PR mentioned below. Kindly review and merge the same. Thanks..!!
https://github.com/Seagate/cortx-k8s/pull/136